src/fiskaltrust.Launcher/Helpers/NumberToStringConverter.cs

```diff
@@ -0,0 +1,42 @@
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace fiskaltrust.Launcher.Helpers
+{
+    public class NumberToStringConverter : JsonConverter<string>
+    {
+        public override bool CanConvert(Type typeToConvert)
+        {
+            return typeof(string) == typeToConvert;
+        }
+        public override string? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.Number)
+            {
+                if (reader.TryGetInt64(out long number))
+                {
+                    return number.ToString(CultureInfo.InvariantCulture);
+                }
+
+                if (reader.TryGetDouble(out var doubleNumber))
+                {
+                    return doubleNumber.ToString(CultureInfo.InvariantCulture);
+                }
+            }
+
+            if (reader.TokenType == JsonTokenType.String)
+            {
+                return reader.GetString();
+            }
+
+            using JsonDocument document = JsonDocument.ParseValue(ref reader);
+            return document.RootElement.Clone().ToString();
```

Comment:
Why is it required to clone the element here?

Reply:
And should we even support this case? What else could it be than a string or a number? 🤔

Reply:
fair point. If we throw on that case the collection still runs through so we can remove it 👌